docs/source/cpp/env_vars.rst

```diff
@@ -85,6 +85,28 @@ that changing their value later will have an effect.
    ``libhdfs.dylib`` on macOS, ``libhdfs.so`` on other platforms).
    Alternatively, one can set :envvar:`HADOOP_HOME`.
 
+.. envvar:: ARROW_S3_LOG_LEVEL
+
+   Controls the verbosity of logging produced by S3 calls. Defaults to Fatal
+   which only produces output in the case of fatal errors. Debug is recommended
+   when you're trying to troubleshoot issues.
+
+   Possible values include:
+
+   - ``Fatal`` (the default)
+   - ``Error``
+   - ``Warn``
+   - ``Info``
+   - ``Debug``
+   - ``Trace``
+   - ``Off``
```

Comment:
How about using all lower case (`fatal`) or all upper case (`FATAL`) instead of capitalized (`Fatal`)?
Because other our environment variables use all lower case or all upper case.

Reply:
Good point, I think a change makes sense. I'm going to go with all uppercase here because that's how I'm used to seeing log levels written out in C++, Java, and Python.

Reply:
Done.